index.bs

```diff
@@ -2397,7 +2397,6 @@ This section contains principles for consideration when designing APIs for devic
 
 <h3 id="device-ids">Use care when exposing identifying information about devices</h3>
 
-
 If you need to give web sites access to information about a device,
 use the guidelines below to decide what information to expose.
```

Comment:
```suggestion
In line with the [Data Minimization](#data-minimization) principle, if you need to give web sites access to information about a device, only expose the minimal amount of data necessary. The guidelines below can help make this determination.```

Reply:
```suggestion
In line with the [Data Minimization](#data-minimization) principle, if you need to give web sites access to information about a device, only expose the minimal amount of data necessary.
```

That last sentence seems a shade redundant (all of these sections provide guidelines that help).